hackathon/templatetags/my_tags.py

```diff
@@ -2,6 +2,8 @@
 # range snippet from: https://www.djangosnippets.org/snippets/1357/
 # adjusted to current project needs based on https://docs.djangoproject.com/en/3.1/howto/custom-template-tags/
 from django.template import Library
+import datetime
+from django.conf import settings
 
 register = Library()
 
```

Comment:
Must have overlooked this earlier, but please add an extra blank line here as well.

Reply:
done